app/src/main/java/fr/free/nrw/commons/upload/DescriptionsAdapter.java

```diff
@@ -122,7 +125,20 @@ public void init(int position) {
                 });
 
             } else {
-                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null, null, null);
+                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null,
+                    getIcon(R.drawable.ic_clear_24dp), null);
+                descItemEditText.setCompoundDrawablePadding(10);
+                descItemEditText.setOnTouchListener((v, event) -> {
```

Comment:
we aren't using [this method](https://developer.android.com/reference/com/google/android/material/textfield/TextInputLayout#setEndIconOnClickListener(android.view.View.OnClickListener))?